kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -2731,6 +2743,105 @@ private void hideSendMessageLayout(boolean hide) {
 
     }
 
+    public void updateSupportGroupTitle(Contact contact) {
+        if (contact != null) {
+            if (!TextUtils.isEmpty(contact.getImageURL())) {
+                try {
+                    if (getContext() != null) {
+                        RequestOptions options = new RequestOptions()
+                                .centerCrop()
+                                .placeholder(R.drawable.applozic_ic_contact_picture_holo_light)
+                                .error(R.drawable.applozic_ic_contact_picture_holo_light);
+
+
+                        Glide.with(getContext()).load(contact.getImageURL()).apply(options).into(toolbarImageView);
+                    }
+                } catch (Exception e) {
+                    e.printStackTrace();
+                }
+            }
+            toolbarTitleText.setText(contact.getDisplayName());
+            switchContactStatus(contact);
+        }
+    }
+
+    public Contact getSupportGroupContact(Channel channel) {
+        List<ChannelUserMapper> userMapperList = ChannelService.getInstance(getContext()).getListOfUsersFromChannelUserMapper(channel.getKey());
+        Utils.printLog(getContext(), "RoleTest", "User Role : " + MobiComUserPreference.getInstance(getContext()).getUserRoleType());
+
+        if (User.RoleType.USER_ROLE.getValue().equals(MobiComUserPreference.getInstance(getContext()).getUserRoleType())) {
+            Map<String, String> metadataMap = channel.getMetadata();
+            if (metadataMap != null) {
+                String conversationAssigne = null;
+                String conversationTitle = null;
+
+                if (metadataMap.containsKey("CONVERSATION_ASSIGNEE")) {
+                    conversationAssigne = metadataMap.get("CONVERSATION_ASSIGNEE");
+                }
+
+                if (metadataMap.containsKey("KM_CONVERSATION_TITLE")) {
+                    conversationTitle = metadataMap.get("KM_CONVERSATION_TITLE");
+                }
+                for (ChannelUserMapper userMapper : userMapperList) {
+                    if (conversationTitle != null && conversationTitle.equals(userMapper.getUserKey())) {
+                        return appContactService.getContactById(conversationTitle);
+                    }
+                }
+                return appContactService.getContactById(conversationAssigne);
+
+            }
+        } else {
```

Comment:
use getListOfUsersFromChannelUserMapper in else part and remove from line number 2769 and then you can check if the user in the group or not then get the contact from db to remove for loop